src/main/resources/templates/repo/kinesis-log-streams.json

```diff
@@ -66,17 +148,42 @@
 		"backupDisabled": true,
 		"tableDescriptor": {
 			"name": "fileHandleAssociationsRecords",
-			"columns": {
-				"timestamp": "timestamp",
-				"associateType": "string",
-				"associateId": "bigint",
-				"fileHandleId": "bigint",
-				"stack": "string",
-				"instance": "string"
+			"description" : "The table contain file handle association records. A FileHandleAssociation record is, a FileHandle (identified by its id) is associated with a Synapse object (identified by its id and type).",

```

Comment:
extraneous comma after "record is".  Extraneous "is" after "(identified by its id)".

> The table contains file handle association records. A FileHandleAssociation record is a FileHandle (identified by its id) along with a Synapse object (identified by its id and type).